tests/test_bot.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+"""
+Test modmail basics
+
+- import module
+- create a bot object
+"""
+from modmail.bot import ModmailBot
+
+
+@pytest.mark.dependency(name="create_bot")
+@pytest.mark.asyncio
+async def test_bot_creation():
+    """Create discord bot."""
+    bot = ModmailBot()
+    assert isinstance(bot, ModmailBot)
+
+    # cleanup
+    await bot.close()
+
+
+@pytest.fixture
+def bot() -> ModmailBot:
+    """
+    Pytest fixture.
+
+    ModmailBot instance
+    """
+    bot: ModmailBot = ModmailBot()
+    return bot
+
+
+@pytest.mark.dependency(depends=["create_bot"])
+@pytest.mark.asyncio
+async def test_bot_aiohttp(bot):
+    """Test aiohttp client session creates and closes without warnings."""
+    import aiohttp
+
+    await bot.create_session()
+    assert isinstance(bot.http_session, aiohttp.ClientSession)
+    await bot.close()
```

Comment:
Rather than this, we can test on bot startup an aiohttp client session is made, and on closing is it closed successfully.

Reply:
As of right now, the bot does not automatically create the aiohttp session.